tests/cpp/ir/type_test.cpp

```diff
@@ -18,19 +18,18 @@ TEST(Type, TypeToString) {
   auto qfl = TypeFactory::get_instance().get_quant_float_type(qi5, qu7, f32);
 
   auto bs1 = TypeFactory::get_instance().get_bit_struct_type(
-      u16, {qi5, qu7}, {0, 5}, {false, false}, {-1, -1}, {{}, {}});
+      u16, {qi5, qu7}, {0, 5}, {-1, -1}, {{}, {}});
```

Comment:
nit: will be nicer to add some comments around the arguments so that readers can know what they mean without looking up the header file ;)

Reply:
done